src/main/java/wenjieBot/Parser.java

```diff
@@ -0,0 +1,65 @@
+package wenjieBot;
+
+import wenjieBot.commands.*;
```

Comment:
Imported classes should be listed explicitly instead of using *, in order to make the class easier to comprehend and maintain.